source/_patterns/01-atoms/svgicon/demo/_svgicon-demo.scss

```diff
@@ -0,0 +1,44 @@
+// All component Sass needs non-printing base config
+@import '../../../00-protons/variables';
+@import '../scss/_icons-generated';
+@import '~font-awesome/scss/mixins';
```

Comment:
Should these already be in the protons non-printing export?

Reply:
protons/_base.scss imports the font-awesome/scss/font-awesome. Those two lines were copied over from the existing icon scss file.

Reply:
I'll hit this in Sass architecture refactor.